thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs

```diff
@@ -86,7 +86,7 @@
           {{#if isDimensionError}}
             <div class="te-form__super-label">... error loading subDimensions for <strong>{{selectedDimension}}</strong></div>
           {{else}}
-            <div class="te-form__super-label">...{{if isDimensionFetchDone 'Displaying' 'Loading'}} top 5 contributing subDimensions for <strong>{{selectedDimension}}</strong></div>
+            <div class="te-form__super-label">...{{if isDimensionFetchDone 'Displaying' 'Loading'}} top {{availableDimensions}} contributing subDimensions for <strong>{{selectedDimension}}</strong></div>
```

Comment:
Let's avoid doing inline css styling here. Maybe have a `span` around `{{selectedDimension}}`. Same applies to line above.

Reply:
<strong> is a valid html tag, however let's stay consistent with the rest of the app and use the same classNames. I believe this styling should already exists for labels (look up `.te-label` in _styles.scss)

Reply:
sure. noted. I forgot to change this instance of the <strong> tag.